scormxblock/scormxblock.py

```diff
@@ -424,7 +432,17 @@ def get_raw_scorm_status(self, request, suffix=''):
         """
         # TODO: handle errors
         # TODO: this is specific to SSLA player at this point.  evaluate for broader use case
-        return Response(self.raw_scorm_status, content_type='application/json', charset='UTF-8')
+        response = Response(self.raw_scorm_status, content_type='application/json', charset='UTF-8')
+        self.update_progress(response.body)
+        return response
+
+    def update_progress(self, scorm_response_body):
```

Comment:
1. Rename function name
2. Please move the function definition below in the file with other progress publishing funcitons. Possibly above `_publish_progress(...)` function.